object_detection/scripts/size_estimator.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+
+# import Locally used packages
+from math import sin, radians
+
+
+
```

Comment:
```suggestion

from math import sin, radians

```

I think the comment here is not needed, and the whitespace can be removed.